kikit/panelize_ui_impl.py

```diff
@@ -252,7 +252,7 @@ def buildLayout(preset, panel, sourceBoard, sourceArea):
                 hbonefirst=layout["hbonefirst"],
                 vbonefirst=layout["vbonefirst"])
             substrates = panel.makeGrid(
-                boardfile=sourceBoard, sourceArea=sourceArea,
+                boardfile=boardList[0].path, sourceArea=sourceArea,
```

Comment:
Just uses the first board in the list.
Would you rather it error out if there are multiple boards in the list?
Could even make a grid for each board in the list?